woof-code/rootfs-skeleton/root/.bashrc

```diff
@@ -1,2 +1,12 @@
 . /etc/profile
 [ -f /etc/bash_completion ] && . /etc/bash_completion
+if [ -x /usr/lib/command-not-found ]; then
+ command_not_found_handle() {
+  if [ -f /var/lib/command-not-found/commands.db ]; then
```

Comment:
This file is created after `update-command-not-found`, and the first `apt update` trigger it. If `command-not-found` is used before `update-command-not-found` runs for the first time, it prints an error message. We don't want this error, because some users don't use apt at all and won't ever `apt update`.